src/App.js

```diff
@@ -1,13 +1,37 @@
-import "./App.css";
-import { Footer, Navbar } from "./frontend/components";
-import { LandingPage } from "./frontend/pages";
+import { useLocation } from "react-router-dom";
+
+import { Footer, Navbar, Sidebar } from "./frontend/components";
 import { Router } from "./frontend/routing/Router";
 
+import "./App.css";
+
+const sidebarRestrict = (pathname) => {
```

Comment:
Not sure what this functionality is for. Was this for private routes? If yes, then I think there are better ways to achieve this via routes?